stable/presto/values.yaml

```diff
@@ -19,6 +19,14 @@ server:
       type: "UseG1GC"
       g1:
         heapRegionSize: "32M"
+  catalog: {}
```

Comment:
Couldn't this have something naming the secret catalog-volume?

Reply:
this is about backward compatibility so that it won't bring break change once upgrade.